### PR TITLE
Add a `Requirements title` question

### PR DESCRIPTION
**Pull request by TheDoubleK**
The first question for building a brief needs to be a title that it can be referred to later.
Words taken from the prototype and agreed with Ralph.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists/questions/briefs/importantDates.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+type: textbox_large
+max_length_in_words: 100
+depends:
+  - "on": "lot"
+    being:
+      - digital-outcomes
+      - digital-specialists
+validations:
+  -
+    name: under_100_words
```
Comment (robyoung) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/importantDates.yml]: Not marked as optional but has no `answer_required` validation.

File: frameworks/digital-outcomes-and-specialists/questions/briefs/expectedOutcomes.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Expected Outcomes
+question: Expected Outcomes
+hint: |
+  Optional
+  
+  What might be delivered, eg a content audit, a discovery document or a notifications system.
```
Comment (robyoung) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/expectedOutcomes.yml]: Could part of this be worked into the question?

Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/expectedOutcomes.yml]: Probably, but this is how it looks in the prototype...

**Comment by robyoung** [comment]
:+1:

**Comment by allait** [comment]
Oh, I just did the same thing with locations (https://github.com/alphagov/digitalmarketplace-frameworks/pull/191), should've checked first

**Pull request merged**